adapters/mqtt-vertx/src/main/java/org/eclipse/hono/adapter/mqtt/VertxBasedMqttProtocolAdapter.java

```diff
@@ -388,18 +385,19 @@ private void publishMessage(final MqttEndpoint endpoint, final String tenantId,
         }
     }
 
-    private Future<String> getRegistrationAssertion(final MqttEndpoint endpoint, final String tenantId, final String logicalDeviceId) {
+    private Future<String> getRegistrationAssertion(final MqttEndpoint endpoint, final String tenantId, final String deviceId) {
         String token = registrationAssertions.get(endpoint);
         if (token != null && !RegistrationAssertionHelperImpl.isExpired(token, 10)) {
             return Future.succeededFuture(token);
         } else {
             registrationAssertions.remove(endpoint);
             Future<String> result = Future.future();
-            getRegistrationAssertion(tenantId, logicalDeviceId).compose(t -> {
+            getRegistrationAssertion(tenantId, deviceId).compose(t -> {
                 // if the client closes the connection right after publishing the messages and before that
                 // the registration assertion has been returned, avoid to put it into the map
                 if (endpoint.isConnected()) {
-                    LOG.trace("caching registration assertion for client [{}]", endpoint.clientIdentifier());
+                    LOG.trace("caching registration assertion for [tenantId: [{}], deviceId: [{}]",
```

Comment:
There is a "[" bracket not closed at the end but are we using "(" brackets around such logs ?

Reply:
No, we are using `[.....]` in logs

Reply:
But having something like "]]" at the end isn't so cool :-) 
I agree on using "[ .. ]" on fields values for example (i.e. tenantId, deviceId, ...) but for representing an "object", wrapping it into different brackets could be more readable.

Reply:
That's why I asked Karsten to remove the nesting so it reads
`LOG.trace("caching registration assertion for [tenantId: {}, deviceId: {}]" ...`

Done